packages/docsify-server-renderer/index.js

```diff
@@ -16,7 +16,7 @@ function cwd(...args) {
 
 function isExternal(url) {
   let match = url.match(
-    /^([^:\/?#]+:)?(?:\/\/([^\/?#]*))?([^?#]+)?(\?[^#]*)?(#.*)?/
+    /^([^:/?#]+:)?(?:\/\/([^/?#]*))?([^?#]+)?(\?[^#]*)?(#.*)?/
```

Comment:
This will be deleted in https://github.com/docsifyjs/docsify/pull/1227

Reply:
Sounds good. I was just fixing the unnecessary escape character. Would you prefer I leave as-is or undo this change?

Reply:
All good, I can resolve the conflict later when it happens